maths/krishnamurthy_number_2.py

```diff
@@ -0,0 +1,81 @@
+import doctest
+
+"""
+A Krishnamurthy number: A number that's equal to the sum of the factorials of its digits
+
+For instance, 145 = 1! + 4! + 5! ; 40585 = 4! + 0! + 5! + 8! + 5!, but 78 != 7! + 8!
+
+For more algorithmic details:
+
+https://mycareerwise.com/programming/category/number-checking/krishnamurthy-number-checking
+"""
+
+
+def factorial(number: int) -> int:
```

Comment:
Just use https://docs.python.org/3/library/math.html#math.factorial or explain why that is not a option..

Reply:
i wanted to build everything from scratch!!!